_docs/interfaces.md

```diff
@@ -4,27 +4,23 @@ title: Interface Configuration
 permalink: /docs/interfaces/
 toc: true
 ---
-Interface classes provide the code that COSMOS uses to receive real-time telemetry from targets and to send commands to targets. The interface that a target uses could be anything (TCP/IP, serial, GPIB, Firewire, etc.), therefore it is important that this is a customize-able portion of any reusable Command and Telemetry System. Fortunately the most common form of interfaces are over TCP/IP sockets, and COSMOS provides interface solutions for these. This guide will discuss how to use these interface classes, and how to create your own.
+Interface classes provide the code that COSMOS uses to receive real-time telemetry from targets and to send commands to targets. The interface that a target uses could be anything (TCP/IP, serial, GPIB, Firewire, etc.), therefore it is important that this is a customizable portion of any reusable Command and Telemetry System. Fortunately the most common form of interfaces are over TCP/IP sockets, and COSMOS provides interface solutions for these. This guide will discuss how to use these interface classes, and how to create your own.
 
 Interfaces have the following methods that must be implemented:
 
 1. **connect** - Open the socket or port or somehow establish the connection to the target.  Note: This method may not block indefinitely.
 1. **connected?** - Return true or false depending on the connection state. Note: This method should return immediately
 1. **disconnect** - Close the socket or port of somehow disconnect from the target.  Note: This method may not block indefinitely.
-1. **read** - Return the next packet of data from the target. Note: This method should block until a packet is available or the interface disconnects.  On a clean disconnect it should return nil.
-1. **write** - Send a packet of data to the target. Note: This method may not block indefinitely.
```

Comment:
You should probably still mention that the 5 methods you deleted from the documentation exist and can be overridden if desired.